lib/d3drm/inc/d3drmdef.h

```diff
@@ -554,6 +554,7 @@ typedef struct _D3DRMVERTEX
 } D3DRMVERTEX, *LPD3DRMVERTEX;
 
 typedef LONG D3DRMGROUPINDEX; /* group indexes begin a 0 */
+static const D3DRMGROUPINDEX D3DRMGROUP_0 = 0;
```

Comment:
So using the `D3DRMGROUPINDEX` typedef is a legacy thing from when I was first decompiling OpenLRR. Some of the gfx functions in Ghidra still have this type in use, but all uses of this type should be replaced with `uint32`, which is what is actually used.

All cases of `D3DRMGROUP_0` should just be replaced with `0`, and all cases of `D3DRMGROUP_ALLGROUPS` should be replaced with `-1` (or `0xffffffff` only if the compiler is complaining about the signed/unsigned difference). The ALLGROUPS one is important to replace, since I don't think the Containers/Mesh modules actually honor this value. Instead `-1` is usually returned on failure.

Can you remove this added line from the `d3drmdef.h`? Keep ALLGROUPS though, just to preserve the file like it originally was.